### ISSUE TITLE
Notification Emails have Logo missing

### ISSUE THREAD

**Issue by shomilsaini** [sct/overseerr#3415]
### Description

If the Application URL is set in General Settings, the Notification Email recieved will have the Logo unresolved. For some reason google cache cannot do the logo.
Sample Logo URL in Mail - https://ci5.googleusercontent.com/proxy/SkC64AUQccA0rMtVRqHLX5v5BvtViWbq0b5w462ZsrzC1qwxEp378SeSNsEPZhkkEiRSf1eA=s0-d-e1-ft#https://overseerr.mywebsite.com/logo_full.png

Sample Test Email -
![image](https://user-images.githubusercontent.com/46788173/231819264-c68dcbbb-f298-4dc6-9a4a-71d253e77a83.png)

### Version

1.32.5

### Steps to Reproduce

1. Go to Settings
2. Click On Application URL
3. Enter valid domain
4. Go to Notifications
5. Send Test Email Notification

### Screenshots

![image](https://user-images.githubusercontent.com/46788173/231819453-0ea6ad28-e317-41af-ae6a-77085eae7e56.png)

### Platform

desktop

### Device

Gmail

### Operating System

Linux

### Browser

Chrome

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by TheCatLady** [comment]
This is unrelated to Overseerr itself, but how you are exposing/proxying it.

**Comment by shomilsaini** [comment]
Hi @TheCatLady,
Thanks for your reponse.
Overseerr is proxied through NPM (NginxProxyManager), I am not sure why this is happening especially because all other images in th email are loding perfectly.
If I go to this URL the Logo is reachable - https://overseerr.mywebsite.com/logo_full.png

Would you be able to point me to what I am doing wrong?

Thanks 😊

**Comment by rock7632** [comment]
Is there a resolution to this issue? I am getting the exact same thing

**Comment by shomilsaini** [comment]
> Is there a resolution to this issue? I am getting the exact same thing

Hi @rock7632,
Yes, I found the resolution.

TL; DR - Go to your proxy/firewall and allow bots with User Agent - "GoogleImageProxy". They need to crawl and fetch your image because the image in the mail is not a direct link to your server. It is Google's cached image.

If you want to know how to do it in a specific scenario, I would have to investigate it. However, I was using Cloudflare which block bots if you enable that setting. So, instead of allowing all bots I added an exception for GoogleImageProxy and the location for this setting is Security > WAF > Firewall Rules.

Cheers!

**Comment by rock7632** [comment]
> > Is there a resolution to this issue? I am getting the exact same thing
> 
> Hi @rock7632, Yes, I found the resolution.
> 
> TL; DR - Go to your proxy/firewall and allow bots with User Agent - "GoogleImageProxy". They need to crawl and fetch your image because the image in the mail is not a direct link to your server. It is Google's cached image.
> 
> If you want to know how to do it in a specific scenario, I would have to investigate it. However, I was using Cloudflare which block bots if you enable that setting. So, instead of allowing all bots I added an exception for GoogleImageProxy and the location for this setting is Security > WAF > Firewall Rules.
> 
> Cheers!

Thanks for your reply. I too am using Cloudflare. Can you tell me what exactly you put in the expression for the firewall rule to allow GoogleImageProxy? Right now if I do a new rule and select Known Bots it shows (cf.client.bot) by default. Thank you

**Comment by shomilsaini** [comment]
> > > Is there a resolution to this issue? I am getting the exact same thing
> > 
> > 
> > Hi @rock7632, Yes, I found the resolution.
> > TL; DR - Go to your proxy/firewall and allow bots with User Agent - "GoogleImageProxy". They need to crawl and fetch your image because the image in the mail is not a direct link to your server. It is Google's cached image.
> > If you want to know how to do it in a specific scenario, I would have to investigate it. However, I was using Cloudflare which block bots if you enable that setting. So, instead of allowing all bots I added an exception for GoogleImageProxy and the location for this setting is Security > WAF > Firewall Rules.
> > Cheers!
> 
> Thanks for your reply. I too am using Cloudflare. Can you tell me what exactly you put in the expression for the firewall rule to allow GoogleImageProxy? Right now if I do a new rule and select Known Bots it shows (cf.client.bot) by default. Thank you

OK, so you could something like this in the screenshot below. This is what I am using as well. Cheers!

![image](https://github.com/sct/overseerr/assets/46788173/540a34ff-545e-4276-8f4a-fd8efeb5fec6)

Let me know if this works for you 😊

**Comment by rock7632** [comment]
> > > > Is there a resolution to this issue? I am getting the exact same thing
> > > 
> > > 
> > > Hi @rock7632, Yes, I found the resolution.
> > > TL; DR - Go to your proxy/firewall and allow bots with User Agent - "GoogleImageProxy". They need to crawl and fetch your image because the image in the mail is not a direct link to your server. It is Google's cached image.
> > > If you want to know how to do it in a specific scenario, I would have to investigate it. However, I was using Cloudflare which block bots if you enable that setting. So, instead of allowing all bots I added an exception for GoogleImageProxy and the location for this setting is Security > WAF > Firewall Rules.
> > > Cheers!
> > 
> > 
> > Thanks for your reply. I too am using Cloudflare. Can you tell me what exactly you put in the expression for the firewall rule to allow GoogleImageProxy? Right now if I do a new rule and select Known Bots it shows (cf.client.bot) by default. Thank you
> 
> OK, so you could something like this in the screenshot below. This is what I am using as well. Cheers!
> 
> ![image](https://user-images.githubusercontent.com/46788173/245307709-540a34ff-545e-4276-8f4a-fd8efeb5fec6.png)
> 
> Let me know if this works for you 😊

This worked for me. Thank you so much for your help. This had me pulling my hair out trying to troubleshoot 👍

**Comment by ghaisasadvait** [comment]
Any way to put a static image or a custom url? My instance is not publicly exposed so the Google image proxy can't reach it
